docs/csharp/whats-new/tutorials/static-abstract-interface-methods.md

```diff
@@ -0,0 +1,147 @@
+
+```csharp
+public record Point<T>(T X, T Y) : IAdditionOperators<Point<T>, Translation<T>, Point<T>>
+    where T : IAdditionOperators<T, T, T>
+```
+
+Finally, when you're performing addition, it's useful to have a property that defines the additive identity value for that type. There's a new experimental interface for that feature: `IAdditiveIdentity<TSelf, TResult>`. A translation of `{0, 0}` is the additive identity: The resulting point is the same as the left operand. The `IAdditiveIdentity<TSelf, TResult>` interface defines one readonly property: `AdditiveIdentity`, that returns the identity value. The `Translation<T>` needs a few changes to implement this interface:
+
+:::code language="csharp" source="./snippets/staticinterfaces/Translation.cs":::
+
+There are a few changes here, so let's walk through them one by one. First, you declare that the `Translation` type implements the `IAdditiveIdentity` interface:
+
+```csharp
+public record Translation<T>(T XOffset, T YOffset) : IAdditiveIdentity<Translation<T>, Translation<T>> 
+```
+
+You next might try implementing the interface member as shown in the following code:
+
+```csharp
+public static Translation<T> AdditiveIdentity =>
+    new Translation<T>(XOffset: 0, YOffset: 0);
+```
+
+The preceding code won't compile, because `0` depends on the type. The answer: Use `IAdditiveIdentity<T>.AdditiveIdentity` for `0`. That change means that your constraints must now include that `T` implements `IAdditiveIdentity<T>`. Now that you've added that constraint on `Translation<T>`, you need to add the same constraint to `Point<T>`:
+
+:::code language="csharp" source="./snippets/staticinterfaces/Point.cs":::
+
+This sample has given you a look at how the interfaces for generic math compose. You wrote a method that relied on the `INumber<T>` interface so that method could be used with any numeric type. You built a class that relies on the addition interfaces to implement a type that only supports one mathematical operation. That type declares its support for those same interfaces so it can be composed in other ways. The algorithms are written using the most natural syntax of mathematical operators.
+
+Experiment with these features and register feedback while they're still in preview. Build generic algorithms that work with any numeric type. Build algorithms using these interfaces where the type argument may only implement a subset of number-like capabilities. You may not build new interfaces that use these capabilities, but you can experiment with using them in your algorithms.
```

Comment:
> Experiment with these features and register feedback while they're still in preview

Where to "register feedback"?
Should there be a link to https://github.com/dotnet/runtime/issues or what's the intended place to file an issue for this?

Reply:
Ping @MadsTorgersen @tannergooding @333fred 

Where should I route people to provide feedback?

Reply:
Got the answer offline.